server/game/AbilityTarget.js

```diff
@@ -79,6 +79,10 @@ class AbilityTarget {
             return [context.player.getOpponent()];
         }
 
+        if(this.choosingPlayer === 'opponent') {
+            return [context.player.getOpponent()];
+        }
+
```

Comment:
You did not need this at the end, but I believe it will be needed in the future so we keep it.